testing/integration/container_cmd_test.go

```diff
@@ -41,7 +42,7 @@ func TestContainerCMD(t *testing.T) {
 	require.NoError(t, err)
 
 	createPolicyReq := kibana.AgentPolicy{
-		Name:        fmt.Sprintf("test-policy-enroll-%d", time.Now().Unix()),
+		Name:        fmt.Sprintf("test-policy-enroll-%s", uuid.New().String()),
```

Comment:
Might be worth creating a helper for this like `UniquePolicyName()` to help people do this properly.